test/integration/test_seq_copy_int.py

```diff
@@ -37,7 +37,7 @@
 _TRAIN_LINE_COUNT_EMPTY = 1
 _DEV_LINE_COUNT = 5
 _TEST_LINE_COUNT = 5
-_TEST_LINE_COUNT_EMPTY = 2
+_TEST_LINE_COUNT_EMPTY = 0
```

Comment:
Isn't this to test that Sockeye handles empty inputs? If so would it be an option to filter sentences before passing them to Sacrebleu instead?

Reply:
true, I'll just pass 0 to the `test_other_clis` invocation then.